src/app/ngrx/reducers/podcast-metrics.reducer.ts

```diff
@@ -54,8 +84,8 @@ export function PodcastMetricsReducer(state: PodcastMetricsModel[] = initialStat
         return newState;
       }
       break;
-    case ActionTypes.CASTLE_PODCAST_ALL_TIME_METRICS_SUCCESS:
-      if (action instanceof CastlePodcastAllTimeMetricsSuccessAction) {
+    case ACTIONS.ActionTypes.CASTLE_PODCAST_ALL_TIME_METRICS_SUCCESS:
```

Comment:
I considered doing a loading/loaded status for all time metrics, but we don't have the podcast seriesId for the load action (chart component that calls the load all time actions is complicated and needs some selector love #141). all time metrics don't cause the same jumpiness in the UI, so I'm ok with letting these fill in as they load.

Reply:
agreed